From 9a04ac451f738b82ada28544c44058c86191b43e Mon Sep 17 00:00:00 2001
From: Chris Morgan <macromorgan@hotmail.com>
Date: Wed, 30 Oct 2024 20:40:53 -0500
Subject: [PATCH 04/10] arm64: dts: rockchip: Add avdd_0v9 and avdd_1v8 for
 RGxx3 HDMI

Define the correct 0.9v and 1.8v regulators for the RGxx3 HDMI port.
The regulators used here appear to power most of the 0.9v and 1.8v
power groups listed on table 3-1 of the rk3566 datasheet.

Signed-off-by: Chris Morgan <macromorgan@hotmail.com>
---
 arch/arm64/boot/dts/rockchip/rk3566-anbernic-rgxx3.dtsi | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3566-anbernic-rgxx3.dtsi b/arch/arm64/boot/dts/rockchip/rk3566-anbernic-rgxx3.dtsi
index f4c753f5d335..b5d7096ffbcc 100644
--- a/arch/arm64/boot/dts/rockchip/rk3566-anbernic-rgxx3.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3566-anbernic-rgxx3.dtsi
@@ -231,6 +231,8 @@ &hdmi {
 	ddc-i2c-bus = <&i2c5>;
 	pinctrl-0 = <&hdmitxm0_cec>;
 	pinctrl-names = "default";
+	avdd-0v9-supply = <&vdda0v9_pmu>;
+	avdd-1v8-supply = <&vcca1v8_pmu>;
 	status = "okay";
 };
 
-- 
2.43.0

